### ISSUE TITLE
NullPointerException: null

### ISSUE THREAD

**Issue by mibby** [kangarko/ChatControl-Pro#1394]
@kangarko Getting double chat with ChatControlPro v8.3.17 and this error when chatting. Downgrading to v8.3.16, all is fine again.

```
[22:10:35] [Async Chat Thread - #0/ERROR]: Could not pass event AsyncPlayerChatEvent to ChatControl v8.3.17
java.lang.NullPointerException: null
	at org.mineacademy.chatcontrol.lib.fo.model.lPt6.const(SourceFile:2593) ~[?:?]
	at org.mineacademy.chatcontrol.lib.fo.model.CoM3.const(SourceFile:1284) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .al.const(SourceFile:174) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .w.const(SourceFile:85) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .w.const(SourceFile:39) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .LPt5.execute(SourceFile:77) ~[?:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.14.4.jar:git-Paper-187]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-187]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-187]
	at net.minecraft.server.v1_14_R1.PlayerConnection.chat(PlayerConnection.java:1713) ~[patched_1.14.4.jar:git-Paper-187]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1628) ~[patched_1.14.4.jar:git-Paper-187]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:41) ~[patched_1.14.4.jar:git-Paper-187]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_222]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
[22:10:35] [Async Chat Thread - #0/ERROR]: Could not pass event AsyncPlayerChatEvent to ChatControl v8.3.17
java.lang.NullPointerException: null
	at org.mineacademy.chatcontrol.lib.fo.model.lPt6.const(SourceFile:2593) ~[?:?]
	at org.mineacademy.chatcontrol.lib.fo.model.CoM3.const(SourceFile:1284) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .A.const(SourceFile:158) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .A.const(SourceFile:86) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .A.const(SourceFile:33) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .LPt5.execute(SourceFile:77) ~[?:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.14.4.jar:git-Paper-187]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-187]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-187]
	at net.minecraft.server.v1_14_R1.PlayerConnection.chat(PlayerConnection.java:1713) ~[patched_1.14.4.jar:git-Paper-187]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1628) ~[patched_1.14.4.jar:git-Paper-187]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:41) ~[patched_1.14.4.jar:git-Paper-187]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_222]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
```

**Comment by kangarko** [comment]
Apology for a late reply, I believe this has been solved in 8.3.18

**Comment by mibby** [comment]
Apologies for the late rely -- seems to be fixed with v8.3.18. Thanks!
